lib/algo/memoization/test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const knapsack = require('./0-1 Knapsack');
+
+class Item {
+  constructor(profit, volume) {
+    this.profit = profit;
+    this.volume = volume;
+  }
+}
+
```

Comment:
I would not use a `class` for returning a plain object w/o any custom functionalities, You can read more about [`constructor vs object literal` on why](https://www.google.com/search?q=constructor%20vs%20object%20literal).

You can can use `function`
```js
function getItem (profit, volume)  {
  return {
    profit,
    volume
  }
}
```